icicle-compiler/src/Icicle/Sea/IO/Zebra.hs

```diff
@@ -174,7 +174,6 @@ seaOfDefReadProgram state = vsep
   , "        /* run compute on the facts read so far */"
   , "        if (*fact_count != 0) {"
   , indent 12 $ vsep $ fmap (\i -> pretty (nameOfCompute i) <+> " (program);") computes
-  , "            *fact_count = 0;"
```

Comment:
so what I mean is to move this `fact_count = 0` to outside the loop in `zebra_read_step`